node/src/components/consensus/protocols/zug.rs

```diff
@@ -2194,15 +2193,19 @@ where
     }
 
     fn request_evidence(&self, peer: NodeId, vid: &C::ValidatorId) -> ProtocolOutcomes<C> {
```

Comment:
As discussed elsewhere, I think this should be renamed `send_evidence` as the name is currently misleading.